tests/py/dynamo/conversion/test_attention.py

```diff
@@ -48,15 +56,16 @@ def forward(self, query, key, value):
             ),
             (
                 "4d-4d",
-                (4, 2, 12, 16),
-                (6, 3, 16, 32),
-                (32, 8, 18, 64),
+                (4, 2, 12, 4),
+                (6, 3, 16, 8),
+                (32, 8, 18, 16),
                 (4, 2, 4, 16),
                 (6, 3, 8, 32),
                 (32, 8, 12, 64),
             ),
         ]
     )
+    @unittest.skip("need to change to custom dynamic shapes")
```

Comment:
Are these SDPA tests skipped because they are added in test_composite_aten_op.py ? If so, can we remove them here ?